$NetBSD: patch-aa,v 1.3 2007/01/06 20:45:45 rillig Exp $

All generated programs must be linked with this flag to find libcob.

--- configure.orig	2005-05-03 11:31:14.000000000 +0200
+++ configure	2007-01-06 21:41:08.000000000 +0100
@@ -25089,7 +25089,7 @@ else
   COB_CFLAGS="-I$includedir"
 fi
 
-COB_LIBS="-L$libdir -lcob ${LIBCOB_LIBS}"
+COB_LIBS="-L$libdir -Wl,-R$libdir -lcob ${LIBCOB_LIBS}"
 COB_CONFIG_DIR="$datadir/$PACKAGE_TARNAME/config"
 COB_LIBRARY_PATH=".:$libdir/$PACKAGE_TARNAME"
 COB_MODULE_EXT=`echo "$shlibext" | sed 's/dll\.a/dll/'`
